gammapy/modeling/models/spatial.py

```diff
@@ -775,6 +775,38 @@ def to_region(self, **kwargs):
             **kwargs,
         )
 
+    @classmethod
+    def from_region(cls, region, **kwargs):
+        """Create a `DiskSpatialModel from a ~regions.EllipseSkyRegion`
+
+        Parameters
+        ----------
+        region : `~regions.EllipseSkyRegion` or ~regions.CircleSkyRegion`
+            region to create model from
+
+        Returns
+        -------
+        spatial_model : `~gammapy.modeling.models.DiskSpatialModel`
+        """
+        if isinstance(region, CircleSkyRegion):
+            region = EllipseSkyRegion(
+                center=region.center, width=region.radius, height=region.radius
+            )
+        if not isinstance(region, EllipseSkyRegion):
+            raise TypeError("Please provide a Circular or Elliptic region")
```

Comment:
It is very good practice to also tell users what they actually passed as input, like:

```suggestion
            raise ValueError(f"Please provide a `CircleSkyRegion` or `EllipseSkyRegion`, got {type(region) instead.}")
```